docs/server-side/v2/introduction/webhook.mdx

```diff
@@ -674,7 +690,7 @@ This event is sent when RTMP streaming and/or browser recording is successfully
         "metadata_id": "********-****-****-****-***********",
         "metadata_timestamp": "2021-11-30T12:58:46.400759043Z",
         "peer_id": "********-****-****-****-***********",
-        "recording_enabled": True,
+        "recording_enabled": true,
```

Comment:
Looks like we're sending these in title case `True/False` (check attached SS)
<img width="1175" alt="Screenshot 2022-11-09 at 11 43 41 AM" src="https://user-images.githubusercontent.com/105427758/200756008-eaa79caf-63ce-4a25-a58c-4b775b19b660.png">

Has this been updated in the backend to be sent as `true/false`? If yes, will it not break any current customer implementations?

Reply:
@Nixon-Samuel can you please check if you are getting `True/False` in all the webhooks because I am getting them as `true/false`

Reply:
@atul107 please check my comment in https://100ms.atlassian.net/browse/HMS-6918

Reply:
Updated all boolean values to standard form